Fighters.cpp

```diff
@@ -0,0 +1,31 @@
+#include <iostream>
+#include "Fighters.h"
+
+Fighters::Fighters(std::string ID,int hp, int dmg) : ID(ID), hp(hp), dmg(dmg) 
+{}
+
+std::string Fighters::getID() const
+{
+    return ID;
+}
+
+int Fighters::getHP() const
+{
+    return hp;
+}
+
+int Fighters::getDmg() const
+{
+    return dmg;
+}
+
+void Fighters::Fight(Fighters &enemy)
+{
+	int newHp = enemy.getHP() - dmg;
+	(newHp > 0) ? enemy.hp = newHp : enemy.hp = 0;
```

Comment:
Miért szükséges ide segédváltozó?

Reply:
A kódolás során egyszerűbbnek tűnt és nem vettük észre, de tényleg meg lehet oldani segédváltozó nélkül is

Reply:
@vizvezetek törölve lett a felesleges segédváltozó!